src/tests/ftest/osa/online_drain.yaml

```diff
@@ -90,3 +90,5 @@ rebuild:
   test_with_rebuild: true
 checksum:
   test_with_checksum: false
+multiple_ranks:
+  rank_list: ["1,2"]
```

Comment:
Now that we're using random, shouldn't this be removed?
```suggestion
```

Reply:
done.